package/snap/snap/snapcraft.yaml

```diff
@@ -87,9 +104,13 @@ parts:
       - python3-distutils
       # For theming
       - adwaita-qt
+      # The QtNetworkAuth module, provided by the Aotu APT repo
+      - libqt5networkauth5
       # PyQt
       - python3-pyqt5
       - python3-pyqt5.qtsvg
+      # PyQt bindings for the QtNetworkAuth module, provided by the Aotu APT repo
+      - python3-pyqt5.qtnetworkauth
```

Comment:
What version are these packages built to? What will the process be to upgrade to a different version of Qt/PyQt?

Reply:
`libqt5networkauth5` is built to match Ubuntu's version of Qt, which is v5.9.5. `python3-pyqt5.qtnetworkauth` is also built to match Ubuntu's PyQt5 version, which for some reason is the slightly newer v5.10.1. The process for upgrading will involve changing the version in makedeb as specified in their `changelog` files, and debugging any new build problems that arise from the new version.